WEB_SERVICE/DNS/_HIGH/DNS_WILDCARD_DOMAINS/meta.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "DNS Wildcard Domains",
+  "short_description": "The security risks associated with DNS wildcard domains, including phishing attacks, subdomain takeover, and the bypass of security policies.",
+  "risk_rating": "high",
```

Comment:
im not sure if this should be high

Reply:
I found that the risk depends on the use case:
- Low Risk: If the wildcard domain is in a controlled environment with strict security policies and proper configuration.
- Medium Risk: If used with weaker controls, exposing potential vulnerabilities.
- High Risk: If misconfigured, leaving subdomains exposed to exploitation, phishing, or data leaks.

Do you think we should lower its risk rating? I tend to go with the highest risk possible.